sql/updates/world/2016_03_19_00_world.sql

```diff
@@ -0,0 +1,3 @@
+-- Blinding Light
+DELETE FROM `spell_script_names` WHERE `spell_id` = 115750;
```

Comment:
It is the preferred method to delete spell scripts by name instead of spell ID.
(I don't remember if it is considered to be part of codestyle or just a rule.)